frontend/src/components/explorer/gemBrowser/MapsAvailable.vue

```diff
@@ -6,12 +6,12 @@
         <span>{{ messages.mapViewerName }}</span>
       </p>
     </header>
-    <div v-if="mapAvailableLimited" class="card-content">
+    <div v-if="mapAvailableLimited" class="card-content pl-3 py-2 pr-2">
```

Comment:
I think this should be `pt-3 px-2 pb2` to match the old style.

Reply:
This component is up for refactoring, so I changed the styling to a `p-2` instead.